resources/views/standings/upload.blade.php

```diff
@@ -1,241 +1,3533 @@
+                }
+
+                supportingVariables.availableDrivers = driver.filter(ele => !jsonResultsDetailsStore.driverID.includes(ele.id));
+                
+                repopulateDriverResultsDropdowns(json, driver, supportingVariables);
+
+                slideUpAllLastRowErrorAlerts(json);
+
+                if(errorIndex <= 2) {
+                    $('#removeRow').removeClass('hover:bg-red-700');
+                    $('#removeRow').addClass('opacity-50 cursor-not-allowed');
+                }
+                
+                disableFirstAndLastReorderBtns();
+
+                checkDuplicateGrid(jsonResultsDetailsStore, supportingVariables);
+                checkGridValueGreaterThanArraySize(jsonResultsDetailsStore, supportingVariables);
+                checkGridValuesStartWith1(jsonResultsDetailsStore, supportingVariables);
+                checkGridValuesForBreakInSequence(jsonResultsDetailsStore, supportingVariables);
+                isAllGridValues0(jsonResultsDetailsStore, supportingVariables);
+
+                isFastestLapPresentAndMatchingWithStatus(json, regexValidationStrings, jsonResultsDetailsStore, supportingVariables);
+                disableToggleBtnsOnTimeError(json, regexValidationStrings, supportingVariables);
+                checkAllTableValuesForErrors(json, regexValidationStrings, jsonResultsDetailsStore, supportingVariables);
+
+                for(let i = 0; i < updatedJSONFromTable.results.length; i++) {
+                    reflectFastestLap(updatedJSONFromTable.results[i].status, points, additionalDetailsStore, supportingVariables, i);
+                }
+            }
+        });
+    }
+
+    function popLastRowValuesFromStores(json, jsonResultsDetailsStore, additionalDetailsStore, supportingVariables) {
+        json.results.pop();
+        supportingVariables.indexPosMap.pop();
+        jsonResultsDetailsStore.driverID.pop();
+        jsonResultsDetailsStore.grid.pop();
+        jsonResultsDetailsStore.stops.pop();
+        jsonResultsDetailsStore.raceTimeInAbsolutes.pop();
+        jsonResultsDetailsStore.raceTimeInIntervals.pop();
+        jsonResultsDetailsStore.originalStatusMinusUnitsPlace.pop();
+        jsonResultsDetailsStore.status.pop();
+        additionalDetailsStore.resultsPoints.pop();
+    }
+
+    function slideUpAllLastRowErrorAlerts(json) {
+        $(`#errorPosAlert${json.results.length}`).slideUp(500);
+        $(`#errorDriverAlert${json.results.length}`).slideUp(500);
+        $(`#errorConstructorAlert${json.results.length}`).slideUp(500);
+        $(`#errorStatusAlert${json.results.length}`).slideUp(500);
+        $(`#errorGridAlert${json.results.length}`).slideUp(500);
+        $(`#errorStopsAlert${json.results.length}`).slideUp(500);
+        $(`#errorFlAlert${json.results.length}`).slideUp(500);
+        $(`#errorTimeAlert${json.results.length}`).slideUp(500);
+    }
+
+    function postJson(json, season) {
+        $.ajax({
```

Comment:
please let jquery die. please end its sufffering move to react

ps. joke comment

Reply:
No more jQuery after this. :)